lib/modules/platform/scm-manager/mapper.ts

```diff
@@ -0,0 +1,23 @@
+import type { Pr as RenovatePr } from '../types';
```

Comment:
```suggestion
import type { Pr } from '../types';
```

do not rename if not needed